packages/specs/schema/src/components/default/genericsMapper.ts

```
@@ -2,69 +1,97 @@
-          generics: undefined
-        });
+      if (next || !refSchema.isClass) {
+        return modelSchema;
       }
 
-      if (type === Object) {
+      return toRef(refSchema, modelSchema, options);
+    }
+
+    if (schema?.has("items")) {
+      const nestedSchema = getNestedSchema(propertyKey, schema.get("items")!, options);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)